### PR TITLE
adding back PUSH_USER mechanism for deploy-image

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

File: hack/deploy-image.sh [hunk 1]
```diff
@@ -2,15 +2,54 @@
...
+        oc login -u "$username" -p "$PUSH_PASSWORD" > /dev/null
+        token=$( oc whoami -t 2> /dev/null || : )
+        oc config use-context "$savectx"
+    else
+        # see if current context has a token
+        token=$( oc whoami -t 2> /dev/null || : )
+        if [ -n "$token" ] ; then
+            username=$( oc whoami )
+        else
+            # get the first user with a token
```
Comment (syedriko) [inline hack/deploy-image.sh:23]: Should we be looking for the first user with a token in the right cluster?

**Comment by syedriko** [comment]
@ewolinetz Just wanted to mention an adjacent PR #509

**Comment by ewolinetz** [comment]
Will document proper "how to" using `oc login` instead

**Pull request closed without merging**